bundles/org.openhab.core.thing/src/main/java/org/openhab/core/thing/binding/ThingHandlerCallback.java

```diff
@@ -57,6 +58,15 @@ public interface ThingHandlerCallback {
      */
     void stateUpdated(ChannelUID channelUID, State state);
 
+    /**
+     * Informs about an update to a historic state for a channel.
+     *
+     * @param channelUID channel UID (must not be null)
```

Comment:
You can remove all "must not be null" in this class, as the class is annotated with `@NonNullByDefault`